File: app/src/main/java/com/simon_eye/cardsscoring/MainActivity.java
package com.simon_eye.cardsscoring;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Find the View that shows the numbers category
        TextView likha = findViewById(R.id.likha);
        TextView tarneeb = findViewById(R.id.tarneeb);

        // Set a click listener on that View
        likha.setOnClickListener(new View.OnClickListener() {
            // The code in this method will be executed when the numbers View is clicked on.
            @Override
            public void onClick(View view) {
                Intent likhaActivity = new Intent(MainActivity.this, com.simon_eye.cardsscoring.LikhaGame.class);
                startActivity(likhaActivity);
            }
        });
        tarneeb.setOnClickListener(new View.OnClickListener() {
            // The code in this method will be executed when the numbers View is clicked on.
            @Override
            public void onClick(View view) {
                Intent tarneebActivity = new Intent(MainActivity.this, com.simon_eye.cardsscoring.tarneebActivity.class);
                startActivity(tarneebActivity);
            }
        });


    }
}

File: app/src/main/java/com/simon_eye/cardsScoring/MainActivity.kt
package com.simon_eye.cardsScoring

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import com.simon_eye.cardsScoring.R.id.likha
import com.simon_eye.cardsScoring.R.id.tarneeb
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Find the View that shows the numbers category
//        val likha = findViewById<TextView>(R.id.likha)
//        val tarneeb = findViewById<TextView>(R.id.tarneeb)

        // Set a click listener on that View
        // The code in this method will be executed when the numbers View is clicked on.
        likha.setOnClickListener {
            val likhaActivity = Intent(this@MainActivity, com.simon_eye.cardsScoring.LikhaGame::class.java)
            startActivity(likhaActivity)
        }
        // The code in this method will be executed when the numbers View is clicked on.
        tarneeb.setOnClickListener {
            val tarneebActivity = Intent(this@MainActivity, com.simon_eye.cardsScoring.TarneebGame::class.java)
            startActivity(tarneebActivity)
        }
    }
}
